src/modules/category/entity/category.database-model.ts

```diff
@@ -0,0 +1,4 @@
+export class CategoryDatabaseModel {
+    id?: number;
```

Comment:
@PCalape Why is this optional?

Reply:
Hello @arnellebalane my bad on this one, this was due to a refactoring move in which I did not continue. This should be required, will change this one. Thanks!

Reply:
Done updating this one @arnellebalane thanks!